api/controllers/album.js

```diff
@@ -1,13 +1,32 @@
 // function: allAlbums
 // sends all parent albums
-export async function allAlbums(req, res) {
+
+/**
+ * /albums : all albums
+ * /albums?name={name} : Search for album that matches this name (fuzzy match)
+ * /albums?tag={tag} : Search for albums that have these tags (fuzzy matches)
+ * /albums?keyword={keyword} : Search for albums that have these keywords (fuzzy matches)
+ * /albums?parentID={parentID} : Search for albums that has this albumId as a parent.
+ * /albums?updated={updated} : Search for albums that were last updated after this date.
+ */
+export async function albumsBy(req, res) {
   let conn;
+  const { name } = req.query;
+  const { tag } = req.query;
+  const { keyword } = req.query;
```

Comment:
can combine these.

Reply:
I keep doing it! Prettier/lint keeps separating them.